dist/svg.js

```diff
@@ -1540,10 +1542,27 @@ SVG.FX = SVG.invent({
       return this
     }
 
+    // set the internal animation pointer at the start position and updates the visualisation
+  , atStart: function() {
+    // Cast boolean to number
+    return this.at(+this.situation.reversed)
+  }
+
+    // set the internal animation pointer at the end position and updates the visualisation
+  , atEnd: function() {
+    // Cast boolean to number
+    return this.at(+!this.situation.reversed)
+  }
+
     // set the internal animation pointer to the specified position and updates the visualisation
   , at: function(pos){
       var durDivSpd = this.situation.duration/this._speed
 
+      // correct position
+      if(pos > 1) pos = 1
+      if(pos < 0) pos = 0
+      if(this.situation.reversed) pos = 1 - pos
```

Comment:
Actually your case was not working in the first place which is why I made this pull request. When you call `at(0.7)` with `reversed:true` it mean that 0.3 of your animation as been done and that is the value you need to use when you modify _start_. Without this line, the position just end up being set to 0.3 and is run backward from there.